### DISCUSSION TITLE
ndarray.cpp in pypi wheel

### DISCUSSION THREAD

**Discussion by ysmu** [wjakob/nanobind#178] [General]
The wheel published to pypi appears to include a `ndarray.cpp` that was later renamed to `nb_ndarray.cpp` in both `v1.1.0` and `v1.1.1`. Perhaps it can be removed?

**Comment by wjakob** [comment]
That's a good catch, thank you for noticing and reporting it—this is a stale file that shouldn't be there. I don't think it is referenced from anything and so should not cause problems. But I will make sure it's not there in future wheels.
